Sources/ProjectDescription/Cache.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+// TODO: Add documentation
+// TODO: Add unit tests
+public struct Cache: Codable, Equatable {
+    public struct Flavor: Codable, Equatable {
```

Comment:
Nitpick: I'm more inclined towards calling this one **profile**. What do you think?

Reply:
Sounds good 👍 Let me rename it then.